### DISCUSSION TITLE
Show-Key Docker Container

### DISCUSSION THREAD

**Discussion by danzfairs** [datalust/seq-tickets#1124] [Help] [answered]
How do you run the seq command when running seq in a docker container?  Whenever I try to run seq, it is trying to run this seq: https://manpages.ubuntu.com/manpages/trusty/man1/seq.1.html

Here is the command I tried:
sudo docker exec -it seq seq -show-key

-it seq refers to the name of my docker instance.

I tried this command from the documentation, but that just stared a new seq container and produced the key for that one.

I was able to connect to my seq instance and print out the Seq.json file and get the key from there by:

1. sudo docker exec -it seq bash
2.  echo $(<"data/Seq.json")

Even though I have the key now, I'm not sure how I can run the backup command

**Comment by nblumhardt** [comment, marked as answer]
Hi! All of the Seq command-line is also exposed by the Docker container (more or less), so it _should_ be possible (if the data volume can be mounted read-only while in use), to run:

```
docker run -v <volume mount> datalust/seq show-key
```

On Docker, however, since there's no equivalent of Windows' DPAPI applied to the value in the config file, it's possible to just copy the value from Seq.json as you've done, and supply this to the `restore` command if needed.
